package.json

```diff
@@ -71,6 +71,7 @@
   "dependencies": {
     "@dsherret/to-absolute-glob": "^2.0.2",
     "code-block-writer": "7.3.0",
+    "conditional-type-checks": "^1.0.0",
```

Comment:
Since this is is imported in the code itself it should be in dependencies

Reply:
That library is only used at compile time (like nameof).

Reply:
OK, I'm reverting that change then, but just in case, I'm seeing it being imported in src/compiler/ast/aliases.ts, src/structures/Structure.ts and others at src/structures.

Reply:
Yeah, that's ok. It goes away at compile time.